docker/php/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM php:7.2-cli
```

Comment:
For the Dockerfile, it can be relevant to use PHP 7.4 IMHO

Reply:
It means we would require PHP 7.4 also in the README, shouldn't we?

Reply:
Not necessarily

Reply:
Fixed